man/man8/zfs.8

```diff
@@ -1654,6 +1784,50 @@ The values of user properties are arbitrary strings, are always inherited, and a
 with the \fBzfs create\fR command set up and enable the swap area using the
 \fBmkswap\fR(8) and \fBswapon\fR(8) commands.  Do not swap to a file on a
 \fBZFS\fR file system. A \fBZFS\fR swap file configuration is not supported.
+.SS "Encryption"
+.LP
+Enabling the \fBencryption\fR feature allows the user to create natively
+encrypted filesystems and volumes. \fBZFS\fR will encrypt all user data
```

Comment:
How about: `Enabling the encryption feature allows the creation of encrypted filesystems and volumes.`